test/table/table_padding.py

```diff
@@ -0,0 +1,304 @@
+from pathlib import Path
+
+import qrcode, pytest
+
+from fpdf import FPDF
+from fpdf.enums import MethodReturnValue, YPos, TableCellFillMode, AlignV
+from fpdf.fonts import FontFace
+from test.conftest import assert_pdf_equal, LOREM_IPSUM
+
+
+def show(pdf):
+    pdf.output(r"c:\data\temp.pdf")
+    import subprocess
+
+    subprocess.Popen(r'explorer "c:\data\temp.pdf"')
+
+
+HERE = Path(__file__).resolve().parent
+IMG_DIR = HERE.parent / "image"
+
+IMAGE = IMG_DIR / "png_images/ba2b2b6e72ca0e4683bb640e2d5572f8.png"
+
+IMAGES_DATA = (
+    (IMAGE, IMAGE, IMAGE, IMAGE),
+    (IMAGE, IMAGE, IMAGE, IMAGE),
+    (IMAGE, IMAGE, IMAGE, IMAGE),
+    (IMAGE, IMAGE, IMAGE, IMAGE),
+)
+
+MULTILINE_TABLE_DATA = (
+    ("Multilines text", "Image"),
+    (LOREM_IPSUM[:200], IMG_DIR / "png_images/ba2b2b6e72ca0e4683bb640e2d5572f8.png"),
+    (LOREM_IPSUM[200:400], IMG_DIR / "png_images/ac6343a98f8edabfcc6e536dd75aacb0.png"),
+    (LOREM_IPSUM[400:600], IMG_DIR / "image_types/insert_images_insert_png.png"),
+    (LOREM_IPSUM[600:800], IMG_DIR / "image_types/circle.bmp"),
+)
+
+TABLE_DATA = (
+    ("First name", "Last name", "Age", "City"),
+    ("Jules", "Smith", "34", "San Juan"),
+    (
+        "Brave Sir Robin ",
+        "the  Not-Quite-So-Brave-As-Sir-Lancelot",
+        "+",
+        "Bridge of Death",
+    ),
+    ("Mary", "Ramos", "45", "Orlando"),
+    ("Carlson", "Banks", "19", "Los Angeles"),
+    ("Lucas", "Cimon", "31", "Angers"),
+)
+
+LONG_TEXT = """
+Professor: (Eric Idle) It's an entirely new strain of sheep, a killer sheep that can not only hold a rifle but is also a first-class shot.
```

Comment:
I ❤️ the Monty Python reference!
https://montypython.fandom.com/wiki/Killer_Sheep